modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceRegexCallbackUrlTestCase.java

```diff
@@ -224,4 +214,29 @@ public void testValidateAccessToken() throws Exception {
 		Assert.assertEquals(valid, "true", "Token Validation failed");
 		EntityUtils.consume(response.getEntity());
 	}
+
+	private ApplicationResponseModel createTestApplication() throws Exception {
+		ApplicationModel application = new ApplicationModel();
```

Comment:
Shall we add the missing new line here?

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096